### PR TITLE
Stops gauss from auto-firing

### PR DESCRIPTION
**Pull request by covertcorvid**
## About The Pull Request
Makes the gauss not automatically fire every couple of seconds until you run dry.

## Why It's Good For The Game
Can choose targets better and use ammo more effectively

## Changelog
:cl:
tweak: Stops gauss gun from automatically firing
/:cl:

### CODE REVIEW

File: nsv13/code/modules/overmap/overmap.dm [hunk 1]
```diff
@@ -473,7 +473,7 @@ Proc to spool up a new Z-level for a player ship and assign it a treadmill.
 		return FALSE
 	if(locate(user) in gauss_gunners) //Special case for gauss gunners here. Takes priority over them being the regular gunner.
 		var/datum/component/overmap_gunning/user_gun = user.GetComponent(/datum/component/overmap_gunning)
-		user_gun.onMouseDown(target)
+		user_gun.onClick(target)
```
Comment (Kmc2000) [inline nsv13/code/modules/overmap/overmap.dm:476]: This is actually based

File: nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/50Cal.dm [hunk 2]
```diff
@@ -50,30 +51,39 @@
...
-	src.special_fx = special_fx
 	src.fx_target = fx_target
 	holder = parent
 	start_gunning()
 
+/datum/component/overmap_gunning/proc/apply_settings(special_fx=FALSE, automatic=FALSE) //Until we have named variables, this should work
+	if(special_fx != null)
+		src.special_fx = special_fx
+	if(automatic != null)
+		src.automatic = automatic
```
Comment (TheFakeElon) [inline nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/50Cal.dm:72]:
```suggestion
	if(!isnull(special_fx))
		src.special_fx = special_fx
	if(!isnull(automatic))
		src.automatic = automatic
```

**Review by Bokkiewokkie — COMMENTED** [review]
Someone said that gauss' safeties are busted because of this.

**Comment by covertcorvid** [comment]
> 
> 
> Someone said that gauss' safeties are busted because of this.

Pre-existing issue but this is fixed now

**Pull request merged**